sidekick_core/test/format_command_test.dart

```diff
@@ -0,0 +1,273 @@
+import 'package:sidekick_core/sidekick_core.dart';
+import 'package:sidekick_core/src/commands/format_command.dart';
+import 'package:sidekick_test/fake_stdio.dart';
+import 'package:sidekick_test/sidekick_test.dart';
+import 'package:test/test.dart';
+
+const String _mainFileUnformatted = '''
+void main() {
+  final test = ['Hello', 'World', 'This', 'is', 'a', 'test', 'for', 'the', 'format', 'command'];
+}
+''';
+const String _mainFileFormatted = '''
+void main() {
+  final test = [
+    'Hello',
+    'World',
+    'This',
+    'is',
+    'a',
+    'test',
+    'for',
+    'the',
+    'format',
+    'command'
+  ];
+}
+''';
+
+void main() {
+  void setupProject(
+    Directory tempDir, {
+    required String pubspecContent,
+    required String mainContent,
+  }) {
+    final pubspec = tempDir.file('pubspec.yaml')..createSync();
+    pubspec.writeAsStringSync(pubspecContent);
+    final mainFile = tempDir.file('lib/main.dart')..createSync(recursive: true);
+    mainFile.writeAsStringSync(mainContent);
+  }
+
+  group('getLineLength', () {
+    late File pubspecYamlFile;
+    late DartPackage package;
+
+    setUp(() {
+      final temp = Directory.systemTemp.createTempSync();
+      pubspecYamlFile = temp.file('pubspec.yaml')..writeAsStringSync('''
+name: dashi
+''');
+      env['SIDEKICK_PACKAGE_HOME'] = temp.path;
+      package = DartPackage.fromDirectory(temp)!;
+
+      addTearDown(() {
+        env['SIDEKICK_PACKAGE_HOME'] = null;
+        temp.deleteSync(recursive: true);
+      });
```

Comment:
I don't think this is required. `getLineLength()` doesn't use `SIDEKICK_PACKAGE_HOME `